tests/functional/u/use/use_maxsplit_arg.txt

```diff
@@ -0,0 +1,19 @@
+use-maxsplit-arg:5:12::Use '1,2,3'.split(',', maxsplit=1)[0] instead
+use-maxsplit-arg:6:11::"Use '1,2,3'[::-1].split(',', maxsplit=1)[0] instead"
```

Comment:
this seems pretty weird @cdce8p , but I have no idea why only this line is formatted as such.

Been scratching my head at this

Reply:
The message itself contains colons which normally are the separator in the test file. The quotes therefore are used to escape them.